tutorials/profiler.md

```diff
@@ -99,7 +104,18 @@ xdg-open /usr/share/ignition/ignition-common3/profiler_vis/index.html
 open /usr/share/ignition/ignition-common3/profiler_vis/index.html
 ```
 
-If you are successful, you should see the profiler output in a browser.
+If you see ``connection error``, there are a couple of things to double check
+1. Is profiler enabled?
```

Comment:
How about:

"Was the profiler enabled when the project you're trying to run was compiled? Note that this isn't the case if you installed Ignition libraries from binaries, for example. You need to compile the project from source with the `ENABLE_PROFILER` variable set."

Reply:
I suggest to add a 4th point. You should configure the environment variable `RMT_PORT` with the right port number.